app/controllers/users.rb

```diff
@@ -0,0 +1,34 @@
+get '/users/new' do
+   <<-HTML
```

Comment:
These HTML strings are never being returned anywhere...  You are creating them but not assigning them to anything.